src/pages/receipts/csvFileReceipt.js

```diff
@@ -161,7 +209,7 @@ const updateInTransitMapping = (shippedBoxes) => {
 const materialTypeMapping = (specimenId) => {
   const tubeId = specimenId.split(' ')[1]
   const materialTypeObject = {'0001':'Serum', '0002':'Serum', '0011':'Serum', '0012':'Serum', '0021':'Serum', 
-                              '0003':  'Whole Bl', '0004': 'Whole Bl', '0005': 'Whole Bl', '0013': 'Whole Bl', '0014' : 'Whole Bl', '0024' : 'Whole Bl',
+                              '0003':  'WHOLE BL', '0004': 'WHOLE BL', '0005': 'WHOLE BL', '0013': 'WHOLE BL', '0014' : 'WHOLE BL', '0024' : 'WHOLE BL',
```

Comment:
Can be better looking.

Reply:
In JavaScript, user cannot directly map the same value to multiple different keys within a single object. Unless I create multiple references/objects to the same value

Reply:
I meant each lines in the object are pushed to right, making code reading not as smooth.